Gemfile.lock

```diff
@@ -10,22 +10,22 @@ GEM
   remote: https://rubygems.org/
   specs:
     eventmachine (1.2.7)
-    faye-websocket (0.10.7)
+    faye-websocket (0.10.9)
       eventmachine (>= 0.12.0)
       websocket-driver (>= 0.5.1)
     mustermann (1.0.3)
-    rack (2.0.7)
-    rack-protection (2.0.5)
+    rack (2.0.8)
+    rack-protection (2.0.7)
       rack
-    sinatra (2.0.5)
+    sinatra (2.0.7)
       mustermann (~> 1.0)
       rack (~> 2.0)
-      rack-protection (= 2.0.5)
+      rack-protection (= 2.0.7)
       tilt (~> 2.0)
-    tilt (2.0.9)
-    websocket-driver (0.7.0)
+    tilt (2.0.10)
+    websocket-driver (0.7.1)
       websocket-extensions (>= 0.1.0)
-    websocket-extensions (0.1.3)
+    websocket-extensions (0.1.4)
```

Comment:
### [websocket-extensions](https://github.com/faye/websocket-extensions-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.1.3 → 0.1.4

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/faye/websocket-extensions-ruby/compare/0.1.3...0.1.4).

These are the individual commits:
- (1a441fa) [Test on Ruby 2.5.0.](https://github.com/faye/websocket-extensions-ruby/commit/1a441fac807e08597ec4b315d4022aea716f3efc)
- (7e64cfb) [Switch license to Apache 2.0.](https://github.com/faye/websocket-extensions-ruby/commit/7e64cfb6ca1e880af619020e4bfac58db51ed242)
- (8ee09b2) [Update Travis target versions.](https://github.com/faye/websocket-extensions-ruby/commit/8ee09b292449082418b359eea196edab30246994)
- (68c1b21) [Fix RSpec warnings about raise_error with no arguments.](https://github.com/faye/websocket-extensions-ruby/commit/68c1b21129ec78193b949b96477132a3c795c7a5)
- (738a0af) [Fix deprecation warning about =~ being called on TrueClass.](https://github.com/faye/websocket-extensions-ruby/commit/738a0af1dd5bbd45b1a912086469a8981cb329ca)
- (249bcc1) [Change markdown formatting of docs.](https://github.com/faye/websocket-extensions-ruby/commit/249bcc106f8bdbcd571ac598af3b9ee896947680)
- (242f007) [Bump version to 0.1.4](https://github.com/faye/websocket-extensions-ruby/commit/242f007cb9cf4eddd33a10216a1ae31e86be42fa)


</details>